### PR TITLE
Fixed unable to build SFML with latest Android NDK

### PR DESCRIPTION
**Pull request by acsbendi**
Thanks a lot for making a contribution to SFML! 🙂

Before you create the pull request, we ask you to check the follow boxes. (For small changes not everything needs to ticked, but the more the better!)

* [ ] Has this change been discussed on [the forum](https://en.sfml-dev.org/forums/index.php#c3) or in an issue before?
* [X] Does the code follow the SFML [Code Style Guide](https://www.sfml-dev.org/style.php)?
* [ ] Have you provided some example/test code for your changes?
* [ ] If you have additional steps which need to be performed list them as tasks!

----

## Description

SFML could not be built using the latest NDK, since CMake did not find any libraries (such as OpenAL).
This PR will fix this.
The problem was caused by the the changes added in NDK r19b. Since then, NDK adds its root path to `CMAKE_FIND_ROOT_PATH`. To tell CMake to also search for the libraries in SFML's directory, we have to add it to `CMAKE_FIND_ROOT_PATH` too.

## Tasks

* [X] Tested on Linux
* [ ] Tested on Windows
* [ ] Tested on macOS
* [ ] Tested on iOS
* [ ] Tested on Android

## How to test this PR?

Try to build SFML using the [latest (r19c) Android NDK](https://developer.android.com/ndk/downloads). Without this PR, it will fail.

### CODE REVIEW

**Comment by auzden** [comment]
This worked for me on NDK r19c on Windows 10. Many thanks!

**Comment by acsbendi** [comment]
@auzden You're welcome :)

File: CMakeLists.txt [hunk 1]
```diff
@@ -44,6 +44,9 @@ endif()
 # project name
 project(SFML)
 
+# avoid missing libraries when building with latest NDK
+set(CMAKE_FIND_ROOT_PATH "${PROJECT_SOURCE_DIR};${CMAKE_FIND_ROOT_PATH}")
+
```
Comment (eXpl0it3r) [inline CMakeLists.txt:49]: As this [seems to only be](https://github.com/android/ndk/issues/912) an issue with the NDK (and potentially other cross-compilations), I'd suggest to put it inside an Android check.

Comment (acsbendi) [reply CMakeLists.txt]: Updated.

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thanks for this! 🙂 

Sorry it took so long 😓

**Comment by acsbendi** [comment]
You're welcome!

**Pull request merged**